### PR TITLE
Self review of trial project

### PR DESCRIPTION
**Pull request by gabrielyea**
# Review of an old project
- corrected linter mistakes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,123 +12,125 @@
...
+
+                    return ar.reduce((a, b) => a + b);}
+
                 </code>
 
-        </section>
-    </div>
+            </section>
+        </div>
     </main>
```
Comment (gabrielyea) [inline index.html:130]: Please, to improve readability separate logical blocks of code with new lines.
You can read good practices here [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 2]
```diff
@@ -12,123 +12,125 @@
...
-    </div>
+            <nav id="navbar">
+                <header class="h1">JS Challenges</header>
+                <a href="#Apple_and_Orange" class="nav-link">Apple and Orange</a>
+                <a href="#Breaking_best_and_worst_records" class="nav-link">Breaking best and worst records</a>
+                <a href="#Cats_and_a_mouse" class="nav-link">Cats and a mouse</a>
+                <a href="#The_hurdle_race" class="nav-link">The hurdle race</a>
+                <a href="#Simple_Array_Sum" class="nav-link">Simple Array Sum</a>
+            </nav>
+        </div>
```
Comment (gabrielyea) [inline index.html:23]: Please don't use the tab key for indentation. Use 2 white spaces as suggested in [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
You can set the number of indentation spaces on your editor settings, or in your formatted configuration file.

**Review by gabrielyea — COMMENTED** [review]
# Please improve readability following the suggestions made in your code.
- use correct indentation
- separate logic blocks of code

**Comment by gabrielyea** [comment]
# Improve the README file
> The readme file needs to improve, which will help your repository look professional. Please use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a starting point for your project.

**Review by SpaYco — APPROVED** [review]
Hi @gabrielyea, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- **NOTE:** your inline comments were present before the summary, which leads to the review being messy, it's always best to keep them together, to do so, when you do an inline comment, you should click on "start review" instead of "add single comment" and it will be grouped with the summary and posted once you submit it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gabrielyea** [comment]
>     * **NOTE:** your inline comments were present before the summary, which leads to the review being messy, it's always best to keep them together, to do so, when you do an inline comment, you should click on "start review" instead of "add single comment" and it will be grouped with the summary and posted once you submit it

Thank you!! I was super confused because the first time I did it I selected the code correctly but I don't know why I wasn't able to do it again!!

**Pull request merged**